model/note/note.go

```diff
@@ -274,6 +282,27 @@ func ensureNotesDir(inst *instance.Instance) (*vfs.DirDoc, error) {
 	return dir, nil
 }
 
+// DebounceMessage is used by the trigger for saving the note to the VFS with a
+// debounce.
+type DebounceMessage struct {
+	NoteID string `json:"note_id"`
+}
+
+func setupTrigger(inst *instance.Instance, fileID string) error {
+	sched := job.System()
+	msg := &DebounceMessage{NoteID: fileID}
+	t, err := job.NewTrigger(inst, job.TriggerInfos{
+		Type:       "@event",
+		WorkerType: "notes-save",
+		Arguments:  fmt.Sprintf("%s:UPDATED:%s", consts.NotesEvents, fileID),
+		Debounce:   "10m",
```

Comment:
10 minutes seems like a very long time. Would performances be so bad with a 1 minute debounce time?

Reply:
Well, @edas has suggested 30 minutes. I think it is a bit too much. I've decreased to 10 minutes to see what it gives, and I will probably decrease another time later if the performances are OK.

Reply:
OK. We'll see then :)